client/src/pages/ProfileSetting/ProfileSetting.tsx

```diff
@@ -0,0 +1,38 @@
+import Grid from '@material-ui/core/Grid';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import useStyles from './useStyles';
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useHistory } from 'react-router-dom';
+import ProfilePhoto from '../../components/ProfilePhoto/ProfilePhoto';
+import { useEffect } from 'react';
+
+export default function ProfileSetting(): JSX.Element {
+  const classes = useStyles();
+
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+
+  const history = useHistory();
+
+  useEffect(() => {
```

Comment:
Why are we initializing the socket here? You can remove it, unless there's a specific reason? :)